### PR TITLE
Moved player health, food, and stamina updates to player.js

### PR DESCRIPTION
**Pull request by TheDudeFromCI**
There currently was no way to update food or foodStamina, and having health packets in the pvp.js file didn't make much sense.

### CODE REVIEW

**Review by rom1504 — CHANGES_REQUESTED** [review]
Player is a type a plugin in flying-squid.
If we create a player plugin then it will contain all plugins

**Comment by rom1504** [comment]
Maybe health.js ?

File: src/lib/plugins/health.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      health: player.health
+    })
+  }
+
+  player.updateHealth = (health) => {
+    player.health = health
+    sendHealthPacket()
+  }
+
+  player.updateFood = (food) => {
```
Comment (rom1504) [inline src/lib/plugins/health.js:15]: can you add that to the doc ?

**Review by rom1504 — APPROVED** [review]

**Comment by rom1504** [comment]
Ah ci is failing

**Comment by Karang** [comment]
> > Ah ci is failing
> 
> Looks like it's failing for an unrelated reason.
> 
> ```
> March 3rd 2021, 12:16:18 [Warning] onItemPlace handler was registered twice for redstone_torch
> Warning: G      March 3rd 2021, 12:16:18 [Warning] onItemPlace handler was registered twice for repeater
> Warning: G      March 3rd 2021, 12:16:18 [Warning] onItemPlace handler was registered twice for oak_sign
> Warning: G      March 3rd 2021, 12:16:18 [Warning] onItemPlace handler was registered twice for spruce_sign
> Warning: G      March 3rd 2021, 12:16:18 [Warning] onItemPlace handler was registered twice for birch_sign
> Warning: G      March 3rd 2021, 12:16:18 [Warning] onItemPlace handler was registered twice for acacia_sign
> Warning: G      March 3rd 2021, 12:16:18 [Warning] onItemPlace handler was registered twice for jungle_sign
> Warning: G      March 3rd 2021, 12:16:18 [Warning] onItemPlace handler was registered twice for dark_oak_sign
> ```

those warnings cannot make ci fail

**Review by rom1504 — APPROVED** [review]

**Pull request merged**